about.html

```diff
@@ -0,0 +1,266 @@
+          </li>
+          <li class="mobile_link">
+            <a href="#join" class="mobile_link_items">Join</a>
+          </li>
+          <li class="mobile_link">
+            <a href="#sponsor" class="mobile_link_items">Sponsor</a>
+          </li>
+          <li class="mobile_link">
+            <a href="#" class="mobile_link_items">News</a>
+          </li>
+          <li class="mobile_link">
+            <a href="#" class="mobile_link_items campaign_actionII"
+              >CC Campaign</a
+            >
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <header class="header headerII">
+      <nav class="authors_social_link">
+        <ul class="authors_profile_container">
+          <li class="authors_profile">
+            <a href="#" class="authors_profile_link"
+              ><img
+                src="./images/facebook-square-brands.jpg"
+                class="social_mid"
+                alt="facebook_profile"
+            /></a>
+          </li>
+          <li class="authors_profile">
+            <a href="#" class="authors_profile_link"
+              ><img
+                src="./images/twitter-square-brands.jpg"
+                class="social_mid"
+                alt="twitter_profile"
+            /></a>
+          </li>
+          <li class="authors_profile">
+            <a href="#" class="authors_profile_link">Logout</a>
+          </li>
+          <li class="authors_profile">
+            <a href="#" class="authors_profile_link">Mypage</a>
+          </li>
+          <li class="authors_profile">
+            <a href="#" class="authors_profile_link">English</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="navbar">
+      <div class="navbar_wrapper">
+        <a href="./index.html" class="link-page">
+          <figure class="logo-img-cont">
+            <img
+              src="./images/university-of-michigan-seeklogo.com.jpg"
+              alt="logo"
+              class="logo_navbar"
+            />
+          </figure>
+        </a>
+        <nav class="link-nav-cont">
+          <ul class="container_list">
+            <li class="destop_content_list">
+              <a href="./index.html" class="destop_content_list_links">Home</a>
+            </li>
+            <li class="destop_content_list">
+              <a href="#main" class="destop_content_list_links">About</a>
+            </li>
+            <li class="destop_content_list">
+              <a href="#join" class="destop_content_list_links">Join</a>
+            </li>
+            <li class="destop_content_list">
+              <a href="#sponsor" class="destop_content_list_links">Sponsor</a>
+            </li>
+            <li class="destop_content_list">
+              <a href="#" class="destop_content_list_links">News</a>
+            </li>
+            <li class="destop_content_list campaign_action">
+              <a href="#" class="destop_content_list_links">CC Campaign</a>
```

Comment:
- [ ] In the original design, this button has a centered orange text, you should consider changing it to match the design better.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142420287-852e9d0d-416e-4c7b-949e-127299cd33e8.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/142420246-7afc7713-275b-449a-befa-c913214ad6f2.png)